From eb47c3f501ebbf9e3b218bb2432d5bdadc04dce1 Mon Sep 17 00:00:00 2001
From: Bogdan Purcareata <bogdan.purcareata@freescale.com>
Date: Tue, 10 Feb 2015 11:08:12 +0000
Subject: [PATCH 05/11] tests: add ppc64 support to the regression live tests

Otherwise The live tests will fail with

"ERROR arch ppc64 not supported"

Send against the working-ppc64 branch.

Signed-off-by: Bogdan Purcareata <bogdan.purcareata@freescale.com>
[PM: added ppc64le]
Signed-off-by: Paul Moore <pmoore@redhat.com>
---
 tests/regression | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/regression b/tests/regression
index 3ab6171..eeb6cfb 100755
--- a/tests/regression
+++ b/tests/regression
@@ -673,7 +673,7 @@ function run_test_live() {
 
 	# setup the arch specific return values
 	case "$arch" in
-	x86|x86_64|x32|arm|aarch64)
+	x86|x86_64|x32|arm|aarch64|ppc64|ppc64le)
 		rc_kill=159
 		rc_allow=160
 		rc_trap=161
-- 
2.3.5

